nexus_constructor/add_component_window.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         initial_edit: bool,
         nx_classes=None,
         tree_view_updater: Callable = None,
+        use_dialogs: bool = False,
```

Comment:
The plan is not use dialogs any more?
If so, just remove the code rather than sticking it behind a flag.

Reply:
There have been a couple of changes that I've made that will be useful for a user to self-select:
 - the camera setting
 - confirmation dialogs

A user may want to have the confirmation when cancelling, particularly for long edits to a component.  The code for the options widget seemed a bit out of scope of this ticket.  So is it still best to delete even if we are keeping it for future implementation?

Reply:
YAGNI.

Reply:
removed :)